person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
I suggest getters and setters should be present for the `rental` since we will be accessing and modifying it here😉.